File: app/src/main/java/by/d1makrat/library_fm/ui/fragment/tabTop/TabTopTracksFragment.java
package by.d1makrat.library_fm.ui.fragment.tabTop;

import by.d1makrat.library_fm.adapter.pages.TabTopTracksAdapter;

public class TabTopTracksFragment extends TabTopItemsFragment<TabTopTracksAdapter> {

    @Override
    protected TabTopTracksAdapter createAdapter() {
        return new TabTopTracksAdapter(getChildFragmentManager());
    }
}

File: app/src/main/java/by/d1makrat/library_fm/ui/fragment/tabTop/TabTopTracksFragment.kt
package by.d1makrat.library_fm.ui.fragment.tabTop

import by.d1makrat.library_fm.adapter.pages.TabTopTracksAdapter

class TabTopTracksFragment : TabTopItemsFragment<TabTopTracksAdapter>() {

    override fun createAdapter(): TabTopTracksAdapter {
        return TabTopTracksAdapter(childFragmentManager)
    }
}
